protoc-gen-go-nmfw/fs/service.go.templ

```diff
@@ -130,6 +130,7 @@ func (s *{{ $svcTypeName }}Service) start(wg *sync.WaitGroup, nc *nats.Conn) {
 			s.Log.Errorf("Encountered an error: %v", natsError)
 			errorsCtr.WithLabelValues(s.Name, "{{ .GoName }}").Inc()
 		},
+		Metadata: map[string]string{},
```

Comment:
hmm, does it really fail without this? I'd say thats perhaps a bug in nats.go, it really should allow empty ones.

Reply:
We do have tests that creates a service without adding it 

https://github.com/nats-io/nats.go/blob/9f57f780070cbba30083c22d0567987c9e0ea1ad/micro/test/service_test.go#L623-L634

let me know what you got without this patch and I'll see if its something we can fix in nats.go

Reply:
Creation works, but using the nats micro cli commands failed without it.

Reply:
You have an example handy? Else I'll check next week - its supposed to be optional so probably a cli/schema bug

Reply:
Ah, figured it out https://github.com/nats-io/nats.go/pull/1242

Reply:
Thanks!